### PR TITLE
Replace `shuffle` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Maybe this should be `shuffle`, and the existing https://lucide.dev/icons?search=shuffle the variation?

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
I'd be open to dropping the existing shuffle icon in favour of this one.

File: icons/shuffle.svg [hunk 1]
```diff
@@ -9,9 +9,9 @@
...
-  <polyline points="16 3 21 3 21 8" />
-  <line x1="4" x2="21" y1="20" y2="3" />
-  <polyline points="21 16 21 21 16 21" />
-  <line x1="15" x2="21" y1="15" y2="21" />
-  <line x1="4" x2="9" y1="4" y2="9" />
+  <path d="M3 18h2c1.1 0 2.5-.7 3.1-1.7l5.8-8.7c.6-.9 2-1.6 3.1-1.6h4" />
+  <polyline points="18 3 21 6 18 9" />
+  <path d="m8.6 8.4-.5-.7C7.5 6.7 6.1 6 5 6H3" />
+  <path d="m13.4 15.6.5.7c.6.9 2 1.7 3.1 1.7h4" />
+  <polyline points="18 15 21 18 18 21" />
```
Comment (karsa-mistmere) [inline icons/shuffle.svg:16]: Fixes wonky curvature + optimizes polyline:
```suggestion
  <path d="M3 18h1.4a4 4 0 0 0 3.296-1.688L13.8 7.7A4 4 0 0 1 17.07 6H21" />
  <path d="m18 3 3 3-3 3" />
  <path d="M3 6h1.9a4 4 0 0 1 3.6 2.25" />
  <path d="M21 18h-3.9a4 4 0 0 1-3.288-1.781L13.3 15.4" />
  <path d="m18 15 3 3-3 3" />
```

**Comment by ericfennis** [comment]
@karsa-mistmere I think I agree.
But the arrows are too small on 100%.

**Comment by danielbayley** [comment]
> arrows are too small on 100%.

@ericfennis Updated…

**Review by karsa-mistmere — APPROVED** [review]
LGTM!

**Review by ericfennis — APPROVED** [review]
Nice work!

**Pull request merged**